components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -595,7 +596,9 @@ private void readChildElements(OMElement serverConfig,
                 } else {
                     log.warn("Choreo System Organization UUID is not defined.");
                 }
-            } 
+            } else if (APIConstants.API_CREATOR_VALIDATION_CONFIGS.equals(localName)) {
+                setApiCreatorValidationConfigurations(element);
+            }
```

Comment:
We are actually validating the backend URL, not the creator right. Shall we rename this and the related other places as well?